source/fundamentals/enterprise-authentication.txt

```diff
@@ -0,0 +1,265 @@
+.. _csharp-enterprise-authentication-mechanisms:
+
+====================================
+Enterprise Authentication Mechanisms
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available only in the {+mongo-enterprise+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
+
+- :ref:`GSSAPI/Kerberos <csharp-kerberos>`
+- :ref:`LDAP (Plain) <csharp-LDAP>`
+
+To authenticate using another mechanism, see the
+:ref:`<csharp-authentication-mechanisms>` fundamentals page. For
+more information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+
+.. _csharp-kerberos:
+
+Authenticate with GSSAPI/Kerberos
```

Comment:
@JamesKovacs can you please remind me whether preparation steps we have in our scripts [here](https://github.com/mongodb/mongo-csharp-driver/blob/master/evergreen/run-gssapi-auth-tests.sh#L22-L45) are required for GSSAPI auth or it can be avoided? Does it make sense to mention them somehow?

Reply:
The steps linked are only needed for EG. Typically the K8s environment would already be configured via a user's login script or done from the command line. Proper configuration of the user's Kerberos environment is outside the scope of what we need to document. We can add it later if users file tickets asking for clarification.